.gitignore

```diff
@@ -1,12 +1,13 @@
 CFHT/
 DECam/
-input/
-output/
+CFHT
+DECam
 test.log
-bin/validateCfht.py
-bin/validateDecam.py
+*.json
```

Comment:
Is it actually necessary to ignore all of this stuff? As it stands, it won't do any harm, but it's just going to cause confusion in future when somebody wants to check a JSON file into the repository. Plus I don't see the benefit at the moment -- it doesn't look like anything you're doing is actually creating JSON files, so it's not likely that they'll accidentally be committed.